src/server/pachyderm_test.go

```diff
@@ -4259,10 +4261,6 @@ func TestGarbageCollection(t *testing.T) {
 		t.Skip("Skipping integration tests in short mode")
 	}
 
-	if os.Getenv(InCloudEnv) == "" {
-		t.Skip("Skipping this test as it can only be run in the cloud.")
-	}
-
 	c := getPachClient(t)
 	require.NoError(t, c.DeleteAll())
 
```

Comment:
Not sure how to comment on a line that's not here ... but the comment on line 4267 I think is no longer valid. The initial garbage collection call + storing the original counts may still be useful though ... since when developing I may run this test as a one off (e.g. `go test -v ./src/server -run=TestGarbageCollection`) and expect it to work regardless of what is in my pachd cluster.